src/js/components/productSupplier/modals/PreferenceTypeModal.jsx

```diff
@@ -14,6 +15,16 @@ const PreferenceTypeModal = ({
   modalData,
   productSupplierId,
 }) => {
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.overflowY = 'hidden';
```

Comment:
Is there no better way to achieve it?

Reply:
Ok I looked into the related issue on the `react-modal` repo and this is a suggested solution, I guess we can live with it...